tools/fip_create/fip_create.c

```diff
@@ -121,6 +122,7 @@ static void print_usage(void)
 	printf("Usage: fip_create [options] FIP_FILENAME\n\n");
 	printf("\tThis tool is used to create a Firmware Image Package.\n\n");
 	printf("Options:\n");
+	printf("\t--plat-toc-flags <val>: 16-bit flag field in toc header\n");
```

Comment:
minor: Clearer to say "16-bit platform specific flag field occupying bits 32-47 in 64-bit toc header".

Also, better to move this under the --dump command as it's not the most important command in the list.